### ISSUE TITLE
[TF2] [Feature Request] Revert Sandman Nerf

### ISSUE THREAD

**Issue by SomeBloodyMinger** [ValveSoftware/Source-1-Games#4778]
Hello TF Team,

I was writing this primarily to see what your opinions would be on reverting the sandman's nerf in Team Fortress 2.

Personally, I never had issues with the weapon myself, and I believe it was mainly only the cleaver combo that people had taken strong issue with. The primary reason I bring this up is because the weapon is, post-nerf, now regarded as one of Scout's worst weapons in the game and is extremely underused. 

I still think the style of the sandman's old stun fit TF2's unique art-style and direction (TF2 never struck me as a serious FPS game where general rules for every other FPS would apply), but I'll leave this entirely up to you. It is just a suggestion after all, but I really do miss this weapon. 

Thanks for listening,
a TF player since 2008.

**Comment by RicochetYT** [comment]
The TF Team tried their hand at stripping a lot of stun mechanics from the game as they are not fun to fight against, I think the cleaver combo getting removed was a side effect or just another nerf. I think bringing back the old stun is a bad idea, as it would go against everything they have worked towards removing, though I think more movement locking mechanics should be removed from the game as well, to complete this move.

**Comment by SomeBloodyMinger** [comment]
I agree that, overall, stun mechanics can be annoying, but I never felt it was 'overpowered' per se, nor did it feel cheap or unfun to fight against when I was restricted, especially when it took skill to hit at effective ranges. 

That brings me to my next point of view: one of the patch notes for the weapon was how it felt 'random' to hit sandman balls most of the time, but I never really got how that was. It was still somewhat effective at mid-range, and at mid range you could still predict and hit your opponent with it at a good rate. 

It always just felt like a lighthearted, non-serious mechanic that takes me back to the TF days where everything wasn't so competitive. Again, this is all a matter of my own personal experience playing TF2, and if the dev team have moved on to updating the game a certain direction then that is 100% their choice. 

Even if it doesn't return, I just hope they keep some semblance of its original design somehow. It was always a fun weapon to me, not one that I would prefer over stock very often, but one I would prefer when I wanted to take a bit of a break from Scout's standard playstyle to have fun with instead.

**Comment by AzureWoof** [comment]
Usually, I wouldn't really consider balance discussions to have a place here. But considering we're finally getting a decently sized update coming soon, which could absolutely have balance changes in it (and we don't really know if we'll ever get another shot at balance changes for the game anytime soon after that), this is definitely an important conversation to have. So, I'll throw my hat into the ring.

My main problem with the nerf in question is how the Sandman + Flying Guillotine combo got completely removed despite it being pretty skill-based at longer ranges, and the fact that the new slow mechanic added to the Sandman is not a fixed value and fluctuates depending on the class you end up slowing, making it difficult to predict how they'll move after being stunned. Not only that, but the slow can still be completely negated by spamming A and D while moving.

The stun removal needed to happen, but the combo potential being completely removed + the slow being inconsistent and barely doing anything is what ruined the weapon, and I wouldn't mind seeing another rework for the weapon this Summer if the update that got announced ends up including balance changes.

In my opinion, the slow mechanic should be changed to make all classes move at a specific fixed movement speed for consistency, along with making the Flying Guillotine's crit on slowed targets, except change its damage to be based on distance. So if you hit a moonshot with the cleaver on a slowed enemy, it will crit. Anything less, and it's a minicrit. Or just make it always crit, but have some kind of reverse damage falloff, kind of like what the Ambassador currently has, except backwards, dealing more damage the further it travels before hitting the slowed target. It's way harder to hit a cleaver on an enemy you've hit with the sandman ball post-nerf because they aren't completely stunned and aren't stuck standing in one place. And the max health penalty currently on the weapon is arguably too punishing for what the weapon does at this point, but I'm not too sure what I'd like to see in place of that.

I do really hope we get a lot of balance changes to the game this summer. There's a ton of weapons in the game that are way too underwhelming to use, and end up just taking up backpack slots with nobody ever using them. It's pretty rare for the game to get large, non-seasonal updates nowadays, and I think this summer update is the perfect opportunity to smooth out some rough edges to put the game in a bit of a better place for a while in case there's another update drought.

**Comment by JoriKos** [comment]
> Reply to https://github.com/ValveSoftware/Source-1-Games/issues/4778#issue-1582722390

No, 150 damage instantly is very strong combined with the awful stun. I'd rather have a weak weapon than something that makes me want to disconnect from the match absolutely instantly. It may not be a serious game or that it was overpowered (I do think it was), but it absolutely does not need to return.

**Comment by FrozenDragon0** [comment]
I think the slowness effect should remain as is on the weapon but I do agree it the effect does need to be changed as it quite literally does nothing on the already slow classes as is and given that the weapon sets your base health to 110 for the upside of a mechanic that is quite useless in it's current state is quite punishing.

**Comment by SomeBloodyMinger** [comment]
> > Reply to [#4778 (comment)](https://github.com/ValveSoftware/Source-1-Games/issues/4778#issue-1582722390)

To be fair, I didn't say it was the combo I was mentioning. It was more the stun mechanic on its own, although I also liked the combo and thought that took skill to use effectively as-well. 

I thought asking for a full reversion WITH the combo would be too much, and a lot of the people I have talked with in-game didn't have an issue with the stun so much as the fact they died instantly to the combo. 

Then again, I also met a lot of people who also had no issue with the stun AND the combo; and others who had issues with both; and so on. TF2's community is more divided on it than you would think. Even I thought I had an unpopular opinion for a while, but when I started asking around I got pretty consistent answers across all groups.

**Comment by jh34ghu43gu** [comment]
Make it function like in mvm against giants where moonshots stun them but they can still shoot.

#3471 is also still an issue

**Comment by SomeBloodyMinger** [comment]
> Usually, I wouldn't really consider balance discussions to have a place here. But considering we're finally getting a decently sized update coming soon, which could absolutely have balance changes in it (and we don't really know if we'll ever get another shot at balance changes for the game anytime soon after that), this is definitely an important conversation to have. So, I'll throw my hat into the ring.
> 
> My main problem with the nerf in question is how the Sandman + Flying Guillotine combo got completely removed despite it being pretty skill-based at longer ranges, and the fact that the new slow mechanic added to the Sandman is not a fixed value and fluctuates depending on the class you end up slowing, making it difficult to predict how they'll move after being stunned. Not only that, but the slow can still be completely negated by spamming A and D while moving.
> 
> The stun removal needed to happen, but the combo potential being completely removed + the slow being inconsistent and barely doing anything is what ruined the weapon, and I wouldn't mind seeing another rework for the weapon this Summer if the update that got announced ends up including balance changes.
> 
> In my opinion, the slow mechanic should be changed to make all classes move at a specific fixed movement speed for consistency, along with making the Flying Guillotine crit on slowed targets, except change its damage to be based on distance. So if you hit a moonshot with the cleaver on a slowed enemy, it will crit. Anything less, and it's a minicrit. Or just make it always crit, but have some kind of reverse damage falloff, kind of like what the Ambassador currently has, except backwards, dealing more damage the further it travels before hitting the slowed target. It's way harder to hit a cleaver on an enemy you've hit with the sandman ball post-nerf because they aren't completely stunned and aren't stuck standing in one place. And the max health penalty currently on the weapon is arguably too punishing for what the weapon does at this point, but I'm not too sure what I'd like to see in place of that.
> 
> I do really hope we get a lot of balance changes to the game this summer. There's a ton of weapons in the game that are way too underwhelming to use, and end up just taking up backpack slots with nobody ever using them. It's pretty rare for the game to get large, non-seasonal updates nowadays, and I think this summer update is the perfect opportunity to smooth out some rough edges to put the game in a bit of a better place for a while in case there's another update drought.

Well RIP. Blog post was changed so that balance changes are almost out of the picture now. Still, it was a good discussion nonetheless.

**Comment by Maggot1970** [comment]
Honestly I never had an issue with the old sandman either. I only think people complained about it when the whole anti-stun mechanic talking points from various Youtubers came in like it was some golden rule for all FPS games that they had to follow. I remember back in 2012, people thought the sandman was useless on its own lol. How the times have changed. 

Please do relook the sandman Valve. It didn't deserve it's fate.

**Comment by JoriKos** [comment]
> Replying to https://github.com/ValveSoftware/Source-1-Games/issues/4778#issuecomment-1431367625

People say the same thing about random crits, that no one complained about it until Uncle Dane made a video, but people have had an issue with both the stun and the combo before any YTer talked about them.

Even if that was the case, doesn't excuse that the weapon was incredibly unfun and bad design.

**Comment by Maggot1970** [comment]
> > Replying to [#4778 (comment)](https://github.com/ValveSoftware/Source-1-Games/issues/4778#issuecomment-1431367625)
> 
> People say the same thing about random crits, that no one complained about it until Uncle Dane made a video, but people have had an issue with both the stun and the combo before any YTer talked about them.
> 
> Even if that was the case, doesn't excuse that the weapon was incredibly unfun and bad design.

But what constitutes 'bad' and 'unfun' design? Again, there are no rules ground into stone that define these terms. They are all just opinion, and you are stating it as if the sandman was objectively unfun and poorly-designed. 

Also, it's funny you bring up random crits. Everyone goes on and on about how random crits are unfair and unfun, and yet when the poll came asking if they should be removed, over 50% of players still liked the crits. 

They were iconic, and fitting for TF2. To remove them would be to remove a part of TF2 itself; and I feel like that is why so many players miss the old sandman. It was iconic, it was fitting, and despite its controversial status towards the end of its lifespan, it still had a lot of players attached to it for the fact it was iconic alone. Myself included, although I still believe it was fair and fun to fight with.

Finally, I don't get what everyone is on about anyway when they say "unfun to fight against." No weapons in TF2 or any FPS for that matter are supposed to be 'fun to fight against. They all have the goal of killing or helping kill enemy players, which thwarts their progress and, with that in consideration, any weapon has the capacity to be 'unfun' to fight against. I don't like the Classic for instance. I hate when I'm killed by a Sniper who has full FOV instantaneously, but I still wouldn't call it overpowered, and everyone still calls it Sniper's worst primary weapon. 

Basically, that is my thoughts behind all of this. I apologise for the long post.

**Comment by JoriKos** [comment]
> Replying to https://github.com/ValveSoftware/Source-1-Games/issues/4778#issuecomment-1434563042

> They are all just opinion, and you are stating it as if the sandman was objectively unfun and poorly-designed.

The Sandman is objectively poorly designed and unfun.

> Also, it's funny you bring up random crits. Everyone goes on and on about how random crits are unfair and unfun, and yet when the poll came asking if they should be removed, over 50% of players still liked the crits.

These polls do not represent the entire TF2 community. Random crits are objectively bad design if you want to strife for a balanced game. No matter how "iconic" they are (ask anyone what's iconic to TF2, random crits are not iconic in the slightest).

> No weapons in TF2 or any FPS for that matter are supposed to be 'fun to fight against.

Fun to fight against doesn't mean you are actively enjoying getting hit by it. I don't feel actively frustrated when fighting a Rocket Launcher or a Scattergun, but when I get hit by the Sandman and my literal ability to play the game is taken away that is not fun.

> I don't like the Classic for instance. I hate when I'm killed by a Sniper who has full FOV instantaneously, but I still wouldn't call it overpowered, and everyone still calls it Sniper's worst primary weapon.

Something doesn't have to be OP to be annoying to fight against. Something can be a weak weapon but very annoying.

TF2 is a quite movement-oriented game, it's not a coincidence that airblast is very often complained about. There may not be any rules around this stuff, but it's pretty obvious that when you take things away from a player in a fight it's not fun. See: Overwatch. It removed a ton of its stun mechanics because people really hated it.

**Comment by Maggot1970** [comment]
> > Replying to [#4778 (comment)](https://github.com/ValveSoftware/Source-1-Games/issues/4778#issuecomment-1434563042)

> The Sandman is objectively poorly designed and unfun.

Source? You say it's objectively-unfun, but that would mean it requires some sort of demonstrable evidence. Also to mention the word "fun" is based as a whole around how one single person with a subjective view perceives their experience. The entire concept of how fun something is, is opinion. Again, you are mistaking opinion and fact. It is my opinion it is neither unfun nor poorly-designed, just as it is your opinion that it is both unfun and poorly-designed. 

> These polls do not represent the entire TF2 community. Random crits are objectively bad design if you want to strife for a balanced game. No matter how "iconic" they are (ask anyone what's iconic to TF2, random crits are not iconic in the slightest).

Valve released a poll, if I remember things correctly, where you were asked your stance on random crits on the main menu. Something EVERY TF2 player has access to at the time of playing. This poll showed support more favouring random crits than showing them as unfavoured. Hence why I mentioned this.


> Fun to fight against doesn't mean you are actively enjoying getting hit by it. I don't feel actively frustrated when fighting a Rocket Launcher or a Scattergun, but when I get hit by the Sandman and my literal ability to play the game is taken away that is not fun.

The problem is, this is all a matter of opinions again. It can be frustrating for me to fight against a certain weapon, but for others it may not be all that bad to deal with. That said, some weapons do have more of a propensity to cause irritation in players I have noticed. But even then, they aren't being nerfed simply because they can cause rage quits. Hell, there is an achievement IN TF2 that you are awarded upon making someone ragequit after dominating them as Pyro. But that said, my point here is there are a TON of unfun weapons to fight against in TF2 that do not need reworking or nerfs at all. Some of the most-complained about weapons include: the phlog, the scorch shot, the force a' nature, the wrap assassin, the sydney sleeper, the CLASSIC, Sniper's primaries as a whole, the stickybomb launcher for multiple reasons, etc. 

All of these weapons which have a higher propensity to be annoying are not nerfed for being annoying at all, so why should the sandman be subject to this rule? It was not INSANELY annoying by any stretch, and the stun mechanic did not last as long as everyone made it out to be. At effective ranges where it was easiest to hit, the stun would apply for anywhere between 1-3 seconds most of the time. Considering you can still dodge left and right to avoid further fire as you are not completely movement-restricted, it really was not that big of an issue. The worst it was against any other class was the Heavy, but the Heavy could hold in his revving minigun whilst stunned, meaning as soon as it wore off, as-long as he still aimed in the Scout's direction, he could kill them easily. 

> Something doesn't have to be OP to be annoying to fight against. Something can be a weak weapon but very annoying.

My point exactly. We shouldn't just nerf weapons because they are annoying. Annoying weapons are already existing in TF2 in lots of different ways. Why should the sandman, something with a unique, iconic mechanic that was not always easy to pull off, be subject to nerfs because it was annoying when it is arguably balanced in the fact it makes you weaker overall and only causes you to excel at what you do good already as Scout, which is picking off single, vulnerable targets? 

If you pulled off a stun against one person in a group of people, it would be relatively useless given your lower health and hence-lower ability to tolerate more people; whilst said-stunned individual is already being guarded by their team. You can maybe use it to aid you in an escape in that situation, but that's about it. 

> TF2 is a quite movement-oriented game, it's not a coincidence that airblast is very often complained about. There may not be any rules around this stuff, but it's pretty obvious that when you take things away from a player in a fight it's not fun. See: Overwatch. It removed a ton of its stun mechanics because people really hated it.

The difference between TF2 and Overwatch is TF2 is nowhere-near as cluttered and chaotic, and doesn't have anywhere near the amount of stun mechanics that Overwatch had. With Overwatch, it was hard to see when someone could stun you, and hence you would be stunned out of nowhere A LOT more than in TF2. 

In TF2, the only way you could be stunned out of nowhere was if you were either the victim of a moonshot, or some Scout who flanked you decided to use the sandman for whatever reason over two simple meatshots. Scout was the only class with this ability, and since so-little Scouts actually used it back then outside of the cleaver combo, you did not feel relentlessly harassed at all with stun mechanics. And if you were afflicted with stun, 9 times out of 10 you saw it coming and had the opportunity to avoid it. Again, it really was not that big of a deal at all. Everytime I was stunned, I had no quarrels with it because either:

a) it took skill and precision to get that hit on me; or
b) it was simply bad luck, being hit from it from a long distance away. I have no problem with bad luck in TF2 given TF2 is just 
    that sort of game where you can die instantly with little you could do to save yourself, depending on where you are, your 
    situation, and many other factors like random crits or if there just so-happen to be stickies or a Pyro with crits around that 
    corner.

**Comment by JoriKos** [comment]
> Replying to https://github.com/ValveSoftware/Source-1-Games/issues/4778#issuecomment-1435516423

> Source?

Yes, Valve. "The feedback on this weapon has been fairly consistent for a while: Players really hate losing the ability to fight back." when they changed it.

> Valve released a poll

They had the question post-match but never released the actual stats, it was all community-made polls.

> my point here is there are a TON of unfun weapons to fight against in TF2 that do not need reworking or nerfs at all.

Do we play the same game? The majority of weapons are not unfun to fight against. Dying may be unfun, but the majority of people who complain about weapons like the Rocket Launcher are just bad.

> We shouldn't just nerf weapons because they are annoying

They should absolutely be changed. Annoying weapons make the game annoying to play and that's the opposite of what TF2 is trying to achieve. The old Sandman was not "iconic" in any way, maybe you just loved the weapon but when people ask what's iconic to TF2 they'll say the characters, cosmetics, and art style. Not "The Sandman and its ability to stun".

> and doesn't have anywhere near the amount of stun mechanics that Overwatch had.

I wonder if there's a reason for that. Maybe because it was one of the most complained about things (especially from TF2 players going to OW) in the game and they removed most of them when they basically decided to redo the game. You can't look at this in any way other than "stun mechanics are not fun".

> With Overwatch, it was hard to see when someone could stun you, and hence you would be stunned out of nowhere A LOT more than in TF2.

That is not true. You could easily see yourself getting stunned and it was just as annoying as not being able to see yourself getting stunned.

The ball wasn't hard to hit and neither was the Cleaver. They are very easy to hit players with and it does an absurd amount of damage for the simple trade-off of having 110 HP. Anyone who says this weapon wasn't a problem is either looking back at it with rose-tinted glasses or doesn't care about balance in the first place. It was a very bad weapon, but the nerfs it received weren't good either. It should be reworked into something different entirely, but it should absolutely not go back to how it used to be.

**Comment by Maggot1970** [comment]
> > Replying to [#4778 (comment)](https://github.com/ValveSoftware/Source-1-Games/issues/4778#issuecomment-1435516423)

> Yes, Valve. "The feedback on this weapon has been fairly consistent for a while: Players really hate losing the ability to fight back." when they changed it.

This does not mean it was objectively unfun. All it meant is a lot of players complained about it. Something that happens with a lot of weapons in TF2, back to my "a lot of weapons are unfun to play against" point.

> They had the question post-match but never released the actual stats, it was all community-made polls.

My bad, I thought I had some sort of memory of seeing a notification on TF2's main page one time. It's very vivid too, with, from what I remember, 54% in support of random crits. Although I don't see any evidence of it online whatsoever... maybe it was a weird fever dream?



> Do we play the same game? The majority of weapons are not unfun to fight against. Dying may be unfun, but the majority of people who complain about weapons like the Rocket Launcher are just bad.

As a Scout player, I always find flamethrowers unfun to fight against. For one, they kill you very quickly considering you have such little health; for two, even if you kill said Pyro, their afterburn will likely finish you off afterward if no health is nearby; for three, they blind you very often when fighting Pyros and they require little precision to get the job done whilst you need to hit every shot past the blinding flames in order to kill the Pyro and MAYBE escape alive if there's a healthkit nearby. 

Also to mention the concept of afterburn is incredibly annoying. A simple particle hits you and you are forced to retreat, as more often than not it will leave you on just above half health as a lightclass. Combine this with continually being lit on fire everytime you try and cross a flank or a checkpoint and needing to retreat over and over again, it gets very annoying. 

With all of this in mind and my dislike of Pyro as a class overall, I still don't think he should be nerfed. He is very annoying to play against for me just because of the amount of hassle they have caused me over my years, but I don't take my hatred of afterburn and let it cloud my judgement. I don't think Pyro needs many nerfs at all aside from a select few I can think of.


> They should absolutely be changed. Annoying weapons make the game annoying to play and that's the opposite of what TF2 is trying to achieve. The old Sandman was not "iconic" in any way, maybe you just loved the weapon but when people ask what's iconic to TF2 they'll say the characters, cosmetics, and art style. Not "The Sandman and its ability to stun".

Again, back to my other argument. Annoying weapons don't equal unfair or unbalanced. I always find the more-annoying weapons to fight against just so happen to be more fun for the user of said-weapon. The Force a Nature is a great example. It gives you great mobility, opens a plethora of opportunities you would not have had before (like tricking a Sniper into thinking you will attack him via the same flank, and then access a height you would not have had access to, to surprise him again), and also gives you the power of extreme knockback, which is always the funnest part of the weapon for me. 

Sending people flying makes you feel so powerful, especially when you can use the knockback combined with environmental aspects, like shooting them up into a wall where they're stuck and then finishing them off with the second shot. This is arguably annoying for the other party. Having a Scout knock you out of a preferred position or stopping you in mid-air completely immobile can be a pain, but again annoyance should not be a factor in determining a weapon's balance. Nonetheless, most Scouts will still prefer the scattergun or even the soda popper before they will pick up the Force a Nature, myself included. 



> I wonder if there's a reason for that. Maybe because it was one of the most complained about things (especially from TF2 players going to OW) in the game and they removed most of them when they basically decided to redo the game. You can't look at this in any way other than "stun mechanics are not fun".

What you missed out here was I compared the differences between TF2 and Overwatch in terms of their stun mechanics. TF2 has had very little from the start of its existence, whereas Overwatch came out with lots of different stun mechanics that were all clunky, and considering Overwatch was an unbalanced mess (even to this day it is), no wonder the stun mechanics were overdone; they were too excessive. In TF2, the only REAL way of stunning players was the sandman. Other classes could affect movement, but the sandman was like no other weapon. And even then the Scout using the sandman was easier to kill than any other Scout, and he needed skill to hit the ball, as it was a projectile. And this "easy to hit" notion was simply not true. The sandman was most-effective at longer ranges, and at longer ranges if you saw the Scout pull out the sandman, you could anticipate a throw and dodge it accordingly. Close range did not stun you entirely, and mid-range stunned you for an ineffective amount of time where you'd honestly be better off getting shots in with your scattergun. 



> That is not true. You could easily see yourself getting stunned and it was just as annoying as not being able to see yourself getting stunned.

When you get stunned knowingly, you can at-least say "Well, I was paying attention and despite my attempts, I was still stunned." You may get a little annoyed, but it's not bad at all given you had the opportunity to avoid the ball and you will need to work on your skill in the future to... well... avoid better. But when the ball comes out of nowhere, often-times it is more annoying in the sense that, from your perspective, you couldn't do anything about it because you didn't know it existed. 

And in Overwatch, at-least from my short experience playing the game at a friend's house one time (I didn't like it), I remember getting stunned and pulled in by Roadhog's chain multiple times over and over again, and most times I did not even see him. This does not apply to the sandman at all. Sure, it can stun you, but it does not then pull you in and also make you an up-close easy kill. Also to mention it wasn't spammable and the recharge time was quite hefty if you weren't picking up your ball everytime after throwing. Finally, the sandman was, again, not used by most good Scouts at all, meaning you would only often see it in the hands of new players who didn't know how to effectively utilise the weapon. In essence, again, it was a fun weapon; not a weapon meant to be taken seriously. And if you ragequit every time you got stunned by it, that isn't a problem with the weapon. That is a problem with the enemy playing against the weapon. It wasn't that hard to counter a sandman Scout, and you are acting as if joining a server with one on the opposing team was immediately disconnect-worthy. 



> The ball wasn't hard to hit and neither was the Cleaver. They are very easy to hit players with and it does an absurd amount of damage for the simple trade-off of having 110 HP. Anyone who says this weapon wasn't a problem is either looking back at it with rose-tinted glasses or doesn't care about balance in the first place. It was a very bad weapon, but the nerfs it received weren't good either. It should be reworked into something different entirely, but it should absolutely not go back to how it used to be.

Here, you are falling into the Sniper syndrome. Where the enemy always sees the success of the Sniper, but does not see the many failures and missed opportunities of the Sniper. Sure, you can see the Scout miss his ball, but oftentimes it does not register that he missed and must now face the consequences of missing; mainly being momentarily disarmed and needing to swap back to a viable weapon, and also having the lower health penalty with no benefit, left wide open. No, a lot of players just registered whenever they went into third person like once every match with a sandman-using Scout and immediately got mad without consideration of what they could be doing better. This is a common coping strategy in TF2 I have noticed, where if they get hit or killed with something they don't like, they'd sooner get mad about it and complain than actually change their playstyle or counter whatever weapon they are playing against. Just be weary and anticipate the Scout pulling out his sandman and dodge accordingly. I did it all the time with zero issues, so I don't get why everyone else seems to have had such a big issue with it?

**Comment by Maggot1970** [comment]
Look, all this said I have realised this argument is only clogging up this page, so I will stop replying now. Feel free to respond one final time, but I think we can agree to disagree on this one for now. Thanks, nonethelessz

**Comment by MikhailaTF** [comment]
A lot has been said already about whichever way a possible re-balance should go and I don't think I can add much to this discussion that hasn't already been said.
However, I would like to add that I too find this to be one of the more "high-priority" weapons when it comes to requiring a rework of any capacity. Currently the negatives of this weapon (less maximum health) far outweighs any of the other attributes on this item, in my opinion.

**Comment by LordExor** [comment]
The Sandman would be fine if they'd get around to fixing the slow mechanic so it actually works.
